ksql-rest-model/src/main/java/io/confluent/ksql/rest/entity/MessageEntity.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class MessageEntity extends KsqlEntity {
+
+  private final Optional<String> message;
+
+  public MessageEntity(
+      @JsonProperty("statementText")  final String statementText,
+      @JsonProperty("warnings")       final List<KsqlWarning> warnings,
```

Comment:
Don't accept `warnings` if you're only passing `null` / empty list.  Just remove the constructor param and pass an empty list to the super class.

Reply:
I don't have warnings for insert into, but I left it in there as MessageEntity could be used by a different endpoint which does produce warnings.

Reply:
It's easy enough for someone to put the warnings parameter back if they need it.  The point is, at the moment its not needed, and I'm not even sure it a warning would get output in the CLI even if one was supplied.

Better to keep the API to want we require _now_.  We may never need to pass a warning via this class...